Assets/Src/Scripts/Input/WebCamSource.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Diagnostics;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Debug = UnityEngine.Debug;
+
+namespace SeedAvatar {
+  public enum DropdownKey {
+    CAMERA = 0,
+    RESOLUTION = 1,
+    FPS = 2,
+    FLIP = 3
+  }
+  public enum ResolutionOptions {
+    LOW = 0,
+    MIDDLE = 1,
+    HIGH = 2,
+  }
+  public enum FpsOptions {
+    LOW = 0,
+    HiGH = 1
+  }
+  public enum FlipOptions {
```

Comment:
```suggestion

  public enum FlipOptions {
```

An empty line between top-level type definitions.